qiskit/dagcircuit/dagcircuit.py

```diff
@@ -38,9 +38,13 @@
 from qiskit.circuit.parameterexpression import ParameterExpression
 from qiskit.dagcircuit.exceptions import DAGCircuitError
 from qiskit.dagcircuit.dagnode import DAGNode, DAGOpNode, DAGInNode, DAGOutNode
+from qiskit.circuit.bit import Bit
 from qiskit.utils.deprecation import deprecate_func
 
 
+BitPosition = namedtuple("BitPosition", ("index", "registers"))
```

Comment:
It's my understanding that using `QuantumCircuit.find_bit` is the current way to get all info about bits and registers. An example of this is in `visualization/circuit/utils.py` in `get_bit_reg_index`. So it is being used with the `QuantumCircuit` version of `find_bit`, and my understanding was also that we were following that model here for `DAGCircuit`.

Reply:
Thanks, I see where this approach comes from then. However, as I understand it,`DAGCircuit` isn't necessarily supposed to be parity with `QuantumCircuit` in terms of its API.

Because we don't seem to have a current use case for finding the registers of a given bit in a DAG (we do it in `DAGCircuit.remove_clbits`, but it's internal and fairly straight-forward), I'd suggest that we don't add support for it speculatively.

Aside, adding a generic `find_bit` method that first checks the instance type of the given bit as a convenience seems odd for a low-level transpiler interface. It arguably makes sense for users of `QuantumCircuit` where programmer convenience is a priority, but relatively few users are authoring quick and dirty transpiler passes at the expense of runtime efficiency.

Reply:
Resolving this since after talking with some folks offline, you're correct that `QuantumCircuit.find_bit` is the preferred method for getting bit locations within registers over `Register.index` 😄. Sorry for the confusion!

My comments on method naming are more just my preference, and it sounds like I might be in the minority there.

Reply:
Ok. Thanks. There's still a lot of work to be done here. I found about a dozen of the 30-some files that need changes. So we'll proceed.